companies/dud-de.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "dud-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "addresses",
+        "ads"
+    ],
+    "name": "Dun & Bradstreet Deutschland GmbH",
+    "address": "Robert-Bosch-Straße 11\n64293 Darmstadt\nDeutschland",
+    "phone": "+49 0615 13800",
+    "fax": "+49 0615 1380360",
+    "email": "info.de@dnb.com",
+    "web": "https://www-prod.dnb.com/de-de/",
+    "sources": [
+        "https://www-prod.dnb.com/de-de/impressum/"
+    ],
+    "request-language": "de",
+    "quality": "verified",
```

```suggestion
    "quality": "verified"
```

Is the comma makeing the bot cause parse errors?